notebooks/Perception_Statistics.ipynb

```diff
@@ -294,13 +295,17 @@
     "from datasetinsights.datasets.synthetic import read_bounding_box_2d\n",
     "\n",
     "bounding_box_definition_id = \"f9f22e05-443f-4602-a422-ebe4ea9b55cb\"\n",
-    "line_width = 5\n",
     "def draw_bounding_boxes(index):\n",
     "    filename = os.path.join(data_root, box_captures.loc[index, \"filename\"])\n",
     "    annotations = box_captures.loc[index, \"annotation.values\"]\n",
+    "\n",
+    "    label_mapping = dict()\n",
+    "    for i in ann_def.get_definition(bounding_box_definition_id)['spec']:\n",
+    "        label_mapping[i['label_id']] = i['label_name']\n",
```

Comment:
Hmm, I guess this broke when the label names were removed from the bboxes. Seems ok for now, but it would be nice to find a cleaner way to do this mapping.